src/Rules/NoModelMakeRule.php

```diff
@@ -27,12 +26,12 @@
  * It is functionally equivalent to simply use the constructor:
  * new User()
  *
- * @implements Rule<MethodCall>
+ * @implements Rule<StaticCall>
```

Comment:
Could you add a test that fails without this change?

Reply:
This is very interesting.

Reply:
Thanks for the feedback on the other items. I'll revise and update. I'll think on how to make a failing test for this one too.